src/openqaoa-azure/openqaoa_azure/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.19"
```

Comment:
how are we doing this versioning?

Reply:
I think this number is currently irrelevant: every time you try to upload it to testPyPi you need to have a new version, so the numbers have been bumped for that reason.

In the final PR we will ever them to the expected.

Actually, probably we will do a release candidate to check that it all works as expected and then do the proper release

Reply:
This page has a great explainer for this: https://peps.python.org/pep-0440/

But yes, this version number is temporary until we do a release candidate on Pypi.